### PR TITLE
Road-134 - Add the quicksuggest2bq DAG

### PR DESCRIPTION
**Pull request by Dexterp37**
This is the last bit required to complete [this ticket](https://mozilla-hub.atlassian.net/browse/ROAD-134).

This schedules a daily job that runs quicksuggest2bq, introduced by mozilla/docker-etl#29.

### CODE REVIEW

File: dags/quicksuggest2bq.py [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    "depends_on_past": False,
+    "start_date": datetime(2021, 11, 18),
+    "email_on_failure": True,
+    "email_on_retry": True,
+    "retries": 2,
+    "retry_delay": timedelta(minutes=30),
+}
+
+project_id = "TODO"
+table_id = "TODO"
```
Comment (Dexterp37) [inline dags/quicksuggest2bq.py:25]: @jklukas what should we put here?

File: dags/quicksuggest2bq.py [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+    quicksuggest2bq = gke_command(
+        task_id="quicksuggest2bq",
+        command=[
+            "python", "quicksuggest2bq/main.py",
+            "--destination-project", project_id,
+            "--destination-table-id", table_id,
+        ],
+        docker_image="gcr.io/moz-fx-data-airflow-prod-88e0/quicksuggest2bq:latest",
+        gcp_conn_id="google_cloud_derived_datasets",
```
Comment (Dexterp37) [inline dags/quicksuggest2bq.py:40]: @jklukas I cargo culted these from the play-store-export, are these correct for this job?

Comment (jklukas) [reply dags/quicksuggest2bq.py:40]: Yes, this all looks fine.

File: dags/quicksuggest2bq.py (hunk 1 above)
Comment (jklukas) [reply dags/quicksuggest2bq.py:25]: Let's make this `moz-fx-data-shared-prod` and `search_terms_derived.remotesettings_suggestions_v1`.

File: dags/quicksuggest2bq.py [hunk 3]
```diff
@@ -0,0 +1,45 @@
+"""
```
Comment (jklukas) [inline dags/quicksuggest2bq.py:1]: How about we name this DAG a bit more generically? Perhaps `contextual_services_import`? There could be other import tasks we would want to run on this schedule.

File: dags/quicksuggest2bq.py [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+Runs a Docker image that imports Quicksuggest suggestions
```
Comment (jklukas) [inline dags/quicksuggest2bq.py:2]: Looks like this old version of the file still exists.

**Review by jklukas — CHANGES_REQUESTED** [review]
Need to remove the old DAG.

File: dags/quicksuggest2bq.py (hunk 4 above)
Comment (Dexterp37) [reply dags/quicksuggest2bq.py]: Whoops.

**Review by jklukas — APPROVED** [review]

**Pull request merged**